lib/pin.rb

```diff
@@ -1,2 +1,17 @@
 class Pin
-end 
+  attr_reader :correct, :position
```

Comment:
I would change the syntax for this to the following.

```
attr_reader :correct,
            :position
```

Reply:
word sounds good I'll make that change